experimentation/build.gradle

```diff
@@ -32,7 +32,12 @@ android {
     }
     lintOptions {
         warningsAsErrors true
-        warning 'GradleDependency'
```

Comment:
Minor (🔍): I think one of those was missed from being deleted as well. See the `lintOptions { ... }` configuration on the `sampletracksapp` app module.